eripheebs/cloudKitTracker
# Issue 6 - Added text input for username and linked to ViewController.…

#### What does this PR do?
Adds ability to enter and view username

#### What are the relevant tickets?
Issue 6

#### Where should the reviewer start?
ViewController.swift

#### How should this be manually tested?
1 - Enter a new username, click 'Set Username'
2 - Verify username updates for record on cloud kit container
